tools/star.wdl

```diff
@@ -180,6 +180,6 @@ task alignment {
         read_one_fastqs: "An array of FastQ files containing read one information"
         read_two_fastqs: "An array of FastQ files containing read two information in the same order as the read one FastQ"
         stardb_tar_gz: "A gzipped TAR file containing the STAR reference files"
-        read_groups: "A string containing the read group information to output in the BAM file"
+        read_groups: "A string containing the read group information to output in the BAM file. Read group fields should be space delimited. Read groups should be comma separated. The ID for each read group must match the basename of a fastq file (up to the first period). Required fields: ID, PU, SM, PL, LB"
```

Comment:
Why did this become required again? Also what's the significance of `ID, PU, SM, PL, LB`? That's not the format my sorting script is expecting or what's passed in the regular `rnaseq_standard`.

Reply:
Picard ValidateSam errors if read groups are not present. Downstream analysis tools such as GATK also require it. So rather than handling special cases, I'd rather the pipeline just require read group information (even if it is just a bunch of "unknown" values. The read group format requires `ID, PU, SM, PL, LB` fields